src/main/java/com/minecolonies/coremod/blocks/BlockHutField.java

```diff
@@ -1,5 +1,6 @@
 package com.minecolonies.coremod.blocks;
 
+import com.minecolonies.api.util.LanguageHandler;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.api.util.LanguageHandler'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)